recipes/adam/build.sh

```diff
@@ -6,8 +6,25 @@ PACKAGE_HOME=$PREFIX/share/$PKG_NAME-$PKG_VERSION-$PKG_BUILDNUM
 mkdir -p $BINARY_HOME
 mkdir -p $PACKAGE_HOME
 
-cp -R * $PACKAGE_HOME
+cp -R distribution/* $PACKAGE_HOME
+# for now, we download the egg file and package it
+# to the make find_adam_egg.sh happy
+cp -R python_egg/*.egg $PACKAGE_HOME
```

Comment:
Can we remove the `find_adam_egg.sh` stuff from `pyadam`?
As far as I can tell, wheels are preferred to eggs now (https://pythonwheels.com/).